lochness/mediaflux/__init__.py

```diff
@@ -148,6 +153,9 @@ def sync_module(Lochness: 'lochness.config',
                                                 BIDS=Lochness['BIDS']) / \
                                                         subpath.parent)
 
+                            mf_local = str(Path(mf_local) / prod['out_dir']) \
```

Comment:
Altering the `mf_local`, which is the target directory to save the data, when `out_dir` is specified for the modality in the configuration file.

Reply:
Linking 
https://github.com/AMP-SCZ/lochness/pull/60/commits/4c201f1a52ca501bc65ca51f77892da253a43d06#r780386000 for my understanding

Reply:
When `out_dir` exists, do we still need the `subpath.parent`?

Reply:
Yes, we still need `subpath.parent` to keep the subdirectory structure of the source data.